refiner-app/Cargo.toml

```diff
@@ -10,7 +10,7 @@ license = "CC0-1.0"
 publish = false
 
 [dependencies]
-near-indexer = { git = "https://github.com/near/nearcore", rev = "04b35b9cc10b9ec94b9436cd943eac14f34f8e2e" }
+near-indexer = { git = "https://github.com/near/nearcore", rev = "313693c442c3081537b51ebc9044800d590aea6b" }
```

Comment:
Note this change conflicts with #20 . I could not use 1.29 or even 1.30-rc4 because neither of those have the rocksdb version update that I need.